cpu/kinetis/periph/gpio.c

```diff
@@ -308,62 +309,75 @@ static inline void irq_handler(PORT_Type *port, int port_num)
     /* take interrupt flags only from pins which interrupt is enabled */
     uint32_t status = port->ISFR;
 
-    for (int i = 0; i < 32; i++) {
-        if ((status & (1u << i)) && (port->PCR[i] & PORT_PCR_IRQC_MASK)) {
-            port->ISFR = (1u << i);
-            int ctx = get_ctx(port_num, i);
-            isr_ctx[ctx].cb(isr_ctx[ctx].arg);
+    while (status) {
+        /* get position of first bit set in status */
+        unsigned pin = bitarithm_lsb(status);
+        /* clear it */
+        status &= ~(1 << pin);
```

Comment:
for consistency you could add a u here, or remove it below.

Reply:
Sorry, I forgot to address this while squashing

Reply:
Let's not tell anyone! ;-)